City-of-Helsinki/drupal-helfi-platform-config
# UHF-9213: Run phpstan

`@todo`

## Things to test
- Make sure `drupal_entity_revision` twig filter works as expected
- Make sure MediaReferenceToObject search api processor works as expected
- Make sure language switcher works as expected
